packages/xo-server-auth-ldap/src/index.js

```diff
@@ -256,10 +256,8 @@ class AuthLdap {
   }
 
   async _authenticate({ username, password }) {
-    const logger = this._logger
-
     if (username === undefined || password === undefined) {
-      logger('require `username` and `password` to authenticate!')
+      logger.error('require `username` and `password` to authenticate!')
```

Comment:
It's not an error, it's a valid condition for which the plugin returns that it does not handle such not credentials.